modules/installation-initializing.adoc

```diff
@@ -169,7 +169,7 @@ endif::rhv[]
 
 * Obtain the {product-title} installation program and the pull secret for your cluster.
 ifdef::restricted[]
-For a restricted network installation, these files are on your bastion host.
+For a restricted network installation, these files are on your mirror registry host.
```

Comment:
The same comment applies here regarding whether the mirror registry host is the same as the installation host.